doc/getting_started_rtkernel.rst

```diff
@@ -243,8 +245,8 @@ The flash usage is text+data, as the RAM initialization values are stored in fla
 
 Run tests on XMC4800 target
 ---------------------------
-In order to compile the test code, enable BUILD_TESTING and disable TEST_DEBUG
-in cmake. Reduce PNET_MAX_FILENAME_LENGTH to 30 bytes.
+In order to compile the test code, enable BUILD_TESTING and disable TEST_DEBUG in
```

Comment:
Isn't this the default (BUILD_TESTING=ON, TEST_DEBUG=OFF)?

Reply:
Changed to "In order to compile the test code, make sure to use BUILD_TESTING and that TEST_DEBUG is disabled. Set PNET_MAX_AR to 1, and reduce PNET_MAX_FILENAME_LENGTH to 30 bytes."